teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
```

Comment:
- [ ] Kindly set the parent permission to true  for this constructor as per requirement